🐙 destination-snowflake-cortex: run up-to-date pipeline [2025-05-17]

airbyteio opened the pull request:
# Update destination-snowflake-cortex

This PR was autogenerated by running `airbyte-ci connectors --name=destination-snowflake-cortex up_to_date --pull`

We've set the `auto-merge` label on it, so it will be automatically merged if the CI pipelines pass.
If you don't want to merge it automatically, please remove the `auto-merge` label.
Please reach out to the Airbyte Connector Tooling team if you have any questions or concerns.


## Operations

- Upgrade the base image to the latest version in metadata.yaml: Successful

- Update versions of libraries in poetry.: Successful

- PATCH bump destination-snowflake-cortex version to 0.2.25: Successful

- Build destination-snowflake-cortex docker image for platform(s) linux/amd64, linux/arm64: Failed

- Create or update pull request on Airbyte repository: Successful

- Add changelog entry: Successful

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/airbytehq/airbyte?pullRequest=51743) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/airbytehq/airbyte?pullRequest=51743) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/airbytehq/airbyte?pullRequest=51743) it.</sub>

airbyteio committed:
Upgrade the base image to the latest version in metadata.yaml
Update versions of libraries in poetry.
PATCH bump destination-snowflake-cortex version to 0.2.25
Create or update pull request on Airbyte repository
Add changelog entry

dbgold17 commented:
/format-fix

octavia-squidington-iii committed:
chore: auto-fix lint and format issues

dbgold17 committed:
Merge branch 'master' into up-to-date/destination-snowflake-cortex

dbgold17 commented:
/run-cat-tests

dbgold17 committed:
Merge branch 'master' into up-to-date/destination-snowflake-cortex

dbgold17 committed:
Merge branch 'master' into up-to-date/destination-snowflake-cortex

dbgold17 committed:
explicitly add jaraco.functools as a dependency

dbgold17 committed:
Merge remote-tracking branch 'refs/remotes/origin/up-to-date/destination-snowflake-cortex' into up-to-date/destination-snowflake-cortex

dbgold17 commented:
/run-cat-tests

dbgold17 committed:
fix pyproject format

dbgold17 commented:
/run-cat-tests

dbgold17 commented:
CATs passing after explicitly requiring `jaraco.functools` I'm good to merge

aaronsteers commented on airbyte-integrations/connectors/destination-snowflake-cortex/metadata.yaml:
```diff
@@ -9,11 +9,11 @@ data:
       - ${embedding.api_base}
   connectorBuildOptions:
-    baseImage: docker.io/airbyte/python-connector-base:3.0.0@sha256:1a0845ff2b30eafa793c6eee4e8f4283c2e52e1bbd44eed6cb9e9abd5d34d844
+    baseImage: docker.io/airbyte/python-connector-base:4.0.0@sha256:d9894b6895923b379f3006fa251147806919c62b7d9021b5cd125bb67d7bbe22
```
The bump from base image 3.0 to 4.0 gives me pause...

Everything else here looks great.

aaronsteers committed:
Merge branch 'master' into up-to-date/destination-snowflake-cortex

aaronsteers commented:
/run-cat-tests

aaronsteers reviewed (approved):
LGTM! 🚀
